File: backends/go-gin/internal/usecase/staff/dto.go
package staff

// UpdateRoleDto はスタッフロール更新のユースケース入力です。
type UpdateRoleDto struct {
	ID         uint
	Role       int
	ExecutorID uint
}

// DestroyDto はスタッフ論理削除のユースケース入力です。
type DestroyDto struct {
	ID         uint
	ExecutorID uint
}

File: backends/kotlin-ktor/src/main/kotlin/com/authorization/usecase/staff/Dto.kt
/*
 * スタッフユースケース DTO モジュール。
 *
 * @author Satoshi Nagashiba <satoshi.nagashiba@gmail.com>
 */
package com.authorization.usecase.staff

/**
 * スタッフロール更新 DTO です。
 *
 * @author Satoshi Nagashiba <satoshi.nagashiba@gmail.com>
 */
data class UpdateRoleDto(
    val id:         Long,
    val role:       Int,
    val executorId: Long,
)

/**
 * スタッフ削除 DTO です。
 *
 * @author Satoshi Nagashiba <satoshi.nagashiba@gmail.com>
 */
data class DestroyDto(
    val id:         Long,
    val executorId: Long,
)
